telco-core/configuration/reference-crs-kube-compare/required/networking/metallb/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
@MarSik How should this reference be checked against cluster manifests.
Its meant to be a stand in for https://github.com/openshift-kni/telco-reference/blob/main/telco-core/configuration/reference-crs/required/networking/metallb/service.yaml

Reply:
No longer used.